man/org.freedesktop.login1.xml

```diff
@@ -630,7 +636,10 @@ node /org/freedesktop/login1 {
       authorization.</para>
 
       <para><function>ScheduleShutdown()</function> schedules a shutdown operation <varname>type</varname> at
-      time <varname>usec</varname> in microseconds since the UNIX epoch. <varname>type</varname> can be one
+      time <varname>usec</varname> in microseconds since the UNIX epoch. Alternatively, if
+      <varname>usec</varname> is zero or <literal>UINT64_MAX</literal> and a maintenance window is
+      configured, <filename>systemd-logind</filename> will use the next time of the maintenance window
```

Comment:
hmm, please do not redefine zero like this, stick to UINT64_MAX. zero should probably mean "asap".

Reply:
IIRC anything less then the current time means ASAP as the timestamp is absolute. Allowing 0 in the dbus interface makes it easier to call it from eg dbus-send. Putting the actually value of UINT64_MAX is just inconvenient :)

Reply:
it's what we do across our codebase these days, we should stick to that

Reply:
meanwhile the --when option of systemctl overloads USEC_INFINITY with "cancel" already o_O

Reply:
worked around